### PR TITLE
Add FAQ about python 3 (#1312)

### PR DESCRIPTION
**Pull request by garyluu**
FAQ about how to run Dockstore 1.4.1 with Python 3.

For issue #1242

requirements3.txt file's branch to be changed

### CODE REVIEW

File: faq.md [hunk 1]
```diff
@@ -135,6 +135,21 @@ Setting the default version affects a number of elements including (but not limi
 4. In the files tab, the default version's files would be displayed first.
 5. The docker pull command reflects the default version
 
+## How to run Dockstore with Python 3?
+
+There are currently issues with avro, cwltool, and Python 3.  See [cwltool](https://github.com/common-workflow-language/cwltool/issues/524) for more info.  To work around this issue, instead of installing avro==1.8.1, install avro-cwl==1.8.3.  Therefore, the pip3 requirements.txt file should end up looking like:
+```
+setuptools==36.5.0
+cwl-runner
+cwltool==1.0.20170828135420
```
[the remark this answers is not in the document]
Comment (garyluu) [reply faq.md]: we can use variable in jekyll, but changing several variables would be even more difficult than changing this section.  Might be better to have a requirements file stored somewhere and then put a link to it.

[the remark this answers is not in the document]
Comment (denis-yuen) [reply faq.md]: Yup, in light of the linked issue, maybe this should be done now.

File: faq.md [hunk 2]
```diff
@@ -135,6 +135,12 @@ Setting the default version affects a number of elements including (but not limi
 4. In the files tab, the default version's files would be displayed first.
 5. The docker pull command reflects the default version
 
+## How to run Dockstore with Python 3?
+
+There are currently issues with avro, cwltool, and Python 3.  See [cwltool](https://github.com/common-workflow-language/cwltool/issues/524) for more info.  To work around this issue, instead of installing avro, install avro-cwl.  Therefore, the pip3 requirements.txt file should end up looking like [this](https://raw.githubusercontent.com/ga4gh/dockstore/feature/1220/requirementsFile/dockstore-client/src/main/resources/requirements3.txt)
```
Comment (denis-yuen) [inline faq.md]: Looks like a 404 (probably due to a merge). 
I think this should probably be a link to file served up by a webservice endpoint and shared with Keiran's request

**Comment by garyluu** [comment]
Closing till everything else is done

**Pull request closed without merging**